addons/web/static/tests/views/fields/html_field_tests.js

```diff
@@ -51,6 +51,32 @@ QUnit.module("Fields", ({ beforeEach }) => {
         assert.strictEqual(target.querySelector(".o_field_html").textContent, "some text");
     });
 
+    QUnit.test('required html field must be set', async function (assert) {
+        await makeView({
```

Comment:
also, we have tooling to lint/prettify our js codebase. You can enable it by running ./addons/web/tooling/enable.sh
Once enabled, the files you edited will be linted and prettified when you do a commit. So you'll have to re-do the commit after enabling the script.